modular_skyrat/modules/cargo/code/items/gbp_punchcard.dm

```diff
@@ -0,0 +1,147 @@
+	icon = 'modular_skyrat/modules/cargo/icons/punchcard.dmi'
+	icon_state = "puncher"
+	w_class = WEIGHT_CLASS_TINY
+
+/obj/machinery/gbp_redemption
+	name = "Good Assistant Points Redemption Machine"
+	desc = "Turn your Good Assistant Points punchcards in here for a payout based on the amount of punches you have, and get a new card!"
+	icon = 'modular_skyrat/modules/cargo/icons/punchcard.dmi'
+	icon_state = "gbp_machine"
+	density = TRUE
+	circuit = /obj/item/circuitboard/machine/gbp_redemption
+
+/obj/machinery/gbp_redemption/wrench_act(mob/living/user, obj/item/tool)
+	. = ..()
+	default_unfasten_wrench(user, tool)
+	return TOOL_ACT_TOOLTYPE_SUCCESS
+
+/obj/machinery/gbp_redemption/attackby(obj/item/attacking_item, mob/user, params)
+	if(default_deconstruction_screwdriver(user, "gbp_machine_open", "gbp_machine", attacking_item))
+		return
+
+	if(default_pry_open(attacking_item, close_after_pry = TRUE))
+		return
+
+	if(default_deconstruction_crowbar(attacking_item))
+		return
+
+	if(istype(attacking_item, /obj/item/gbp_punchcard))
+		var/obj/item/gbp_punchcard/punchcard = attacking_item
+		var/amount_to_reward = punchcard.punches * 50
+		if(!punchcard.punches)
+			playsound(src, 'sound/machines/scanbuzz.ogg', 100)
+			say("You can't redeem an unpunched card!")
+			return
+		var/obj/item/card/id/card_used
+		if(isliving(user))
+			var/mob/living/living_user = user
+			card_used = living_user.get_idcard(TRUE)
+		if(card_used?.registered_account)
+			playsound(src, 'sound/machines/printer.ogg', 100)
+			card_used?.registered_account.adjust_money(amount_to_reward, "GAP: [punchcard.punches] punches")
+			say("Rewarded [amount_to_reward] to your account, and dispensed a ration pack! Thank you for being a Good Assistant! Please take your new punchcard.")
+			new /obj/item/storage/fancy/nugget_box(get_turf(src))
+			new /obj/item/gbp_punchcard(get_turf(src))
+			qdel(attacking_item)
+			return
+		else
+			playsound(src, 'sound/machines/scanbuzz.ogg', 100)
+			say("You cannot redeem a punchcard without an ID card with a valid account!")
+			return
+	return ..()
+
+/obj/item/circuitboard/machine/gbp_redemption
+	name = "Good Assistant Point Redemption Machine"
+	greyscale_colors = CIRCUIT_COLOR_SUPPLY
+	build_path = /obj/machinery/gbp_redemption
+	req_components = list(
+		/datum/stock_part/servo = 1)
+
+
+/datum/outfit/job/rd
+	backpack_contents = list(
+		/obj/item/melee/baton/telescopic = 1,
+		/obj/item/gbp_puncher = 1
+	)
+
+/datum/outfit/job/hos
+	backpack_contents = list(
+		/obj/item/evidencebag = 1,
+		/obj/item/gbp_puncher = 1,
+	)
+
+/datum/outfit/job/hop
+	backpack_contents = list(
+		/obj/item/melee/baton/telescopic = 1,
+		/obj/item/gbp_puncher = 1,
+	)
+
+/datum/outfit/job/ce
+	backpack_contents = list(
+		/obj/item/melee/baton/telescopic = 1,
+		/obj/item/construction/rcd/ce = 1,
+		/obj/item/gbp_puncher = 1,
+	)
+
+/datum/outfit/job/cmo
```

Comment:
```
/datum/outfit/job/cmo/pre_equip(mob/living/carbon/human/H, visualsOnly)
    . = ..()
    backpack_contents += list(/obj/item/gbp_puncher = 1)
```
You don't need to override the whole list, then we'd have to update it every time TG changes. Just add to it in pre_equip